backend/dal/Migrations/20211110022507_v1.0.14.2.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Are these related to the bug? I am guessing the migrations are being piggybacked.

Reply:
This is related to the bug yes, the newer view provides the pid in the expected format.

Reply:
Previously the view only exposed the pid as an int, and there isn't a lot you can do in cql to get around that.